assignments/prototypes.js

```diff
@@ -22,6 +30,19 @@
   * takeDamage() // prototype method -> returns the string '<object name> took damage.'
   * should inherit destroy() from GameObject's prototype
 */
+function CharacterStats(attr) {
+  GameObject.call(this, attr);
+  this.hp = attr.hp;
+  this.name = attr.name;
+}
+
+// chain CharacterStats' prototype to GameObject's prototype
+CharacterStats.prototype = Object.create(GameObject.prototype);
```

Comment:
Also add `CharacterStats.prototype.constructor = CharacterStats`. By default it starts that way when you create a constructor function but when we overwrite the prototype with the base function prototype it changes, so we need to change it back so that the `instanceof` operator works as intended.